multimedia/ffmpeg-devel/Portfile

```diff
@@ -119,6 +119,9 @@ patchfiles-append   patch-libavcodec-librsvgdec.diff
 # https://trac.macports.org/ticket/69678
 # patchfiles-append   patch-fix-vulkan.diff
 
+#add retina resolution capabilities via -movflags write_pixeldensity flag
+patchfiles-append   patch-add-pixeldensity.diff
```

Comment:
Has this been tested on systems which do not support Retina displays?

Reply:
it won't affect you at all unless you deliberately invoke the -movflags write_pixeldensity flags. and if you do so, i don't think it will make a difference.